internal/service/advancedclustertpf/resource_schema.go

```diff
@@ -307,28 +375,52 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 				},
 			},
 			"root_cert_type": schema.StringAttribute{
-				Computed:            true,
-				Optional:            true,
+				Computed: true,
+				Optional: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
 				MarkdownDescription: "Root Certificate Authority that MongoDB Cloud cluster uses. MongoDB Cloud supports Internet Security Research Group.",
 			},
 			"state_name": schema.StringAttribute{
-				Computed:            true,
+				Computed: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
 				MarkdownDescription: "Human-readable label that indicates the current operating condition of this cluster.",
 			},
-			"tags": schema.MapAttribute{ // TODO: was ListNestedAttribute, changed to align with flex cluster, might be breaking change
-				ElementType:         types.StringType,
+			// TODO: We want to avoid breaking changes even though it is incompatible with flex cluster and project resource
```

Comment:
is this comment still true about incompatible with flex and project?

Reply:
Yes, see [flex](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/CLOUDP-262066-flex-cluster-dev/internal/service/flexcluster/resource_schema.go#L69) which is a map

Reply:
ah, ok